File: src/main/java/quizapi/payload/QuestionPayload.java
package quizapi.payload;

import java.util.Collection;
import java.util.List;

public class QuestionPayload {
    public Long id;

    public CategoryPayload category;

    public String description;

    public String difficulty;

    public List<AnswerPayload> answers;

    public QuestionPayload() {
    }

    public QuestionPayload(Long id, CategoryPayload category, String description, String difficulty, List<AnswerPayload> answers) {
        this.id = id;
        this.category = category;
        this.description = description;
        this.difficulty = difficulty;
        this.answers = answers;
    }
}

File: src/main/kotlin/com/shkila/quizapi/payload/QuestionPayload.kt
package com.shkila.quizapi.payload

data class QuestionPayload(
        var id: Long? = null,
        var category: CategoryPayload? = null,
        var description: String? = null,
        var difficulty: String? = null,
        var answers: List<AnswerPayload>? = null
)
